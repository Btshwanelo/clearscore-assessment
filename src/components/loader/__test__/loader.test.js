import React from "react";
import Enzyme from "enzyme";
import { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

import Loader from "../index";

Enzyme.configure({ adapter: new Adapter() });

describe("Loader component", () => {
  it("should render correctly", () => {
    const component = shallow(<Loader isLoading={true} />);

    expect(component.debug()).toMatchSnapshot();
  });
});
