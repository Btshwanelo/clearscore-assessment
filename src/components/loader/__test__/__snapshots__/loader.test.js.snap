// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render correctly 1`] = `
"<div>
  <ForwardRef(Backdrop) sx={{...}} open={true}>
    <ForwardRef(CircularProgress) color=\\"inherit\\" />
  </ForwardRef(Backdrop)>
</div>"
`;
